dhcpv4/option_userclass.go

```diff
@@ -32,6 +36,9 @@ func (op *OptUserClass) ToBytes() []byte {
 // Length returns the option length
 func (op *OptUserClass) Length() int {
 	ret := 0
+	if !op.Rfc3004 && len(op.UserClasses) == 1 {
```

Comment:
the `String` method should be modified as well to only print the first user class only

Reply:
If more than 1 user class is present, I am assuming rfc3004 is being used so printing all of them should be fine.